test/unit/quicknxs/interfaces/data_handling/test_processing_workflow.py

```diff
@@ -0,0 +1,57 @@
+import os
+from copy import deepcopy
+
+import pytest
+from orsopy.fileio import load_orso
+
+from quicknxs.interfaces.configuration import Configuration
+from quicknxs.interfaces.data_handling.processing_workflow import DEFAULT_OPTIONS, ProcessingWorkflow
+from quicknxs.interfaces.data_manager import DataManager
+
+
+@pytest.mark.datarepo
+def test_orso_output(data_server, tmpdir):
+    """Test saving reflectivity curves to ORSO"""
+    conf = Configuration()
+    conf.cut_first_n_points = 0
```

Comment:
Make a call to `Configuration.setup_default_values()` in case a previous test messed the values of the class variables.
`Configuration.setup_default_values()` is listed at the end of the declaration of `class Configuration` so few developers are aware of its existence. Can you please move its location to right after the declaration of the class variables?